### DISCUSSION TITLE
Line break between \left and \right

### DISCUSSION THREAD

**Discussion by HughGrovesArup** [KaTeX/KaTeX#3457] [Q&A] [answered]
I'm creating equation using KaTeX in in-line mode. Generally, the automatic line breaking works well, apart from when an equation is enclosed in brackets.

Is there a workaround to allow line breaks within bracketed expressions?

**Comment by edemaine** [comment, marked as answer]
This is how LaTeX works, I'm afraid: line breaks are possible only at the top level.

A workaround is to use `\big`, `\bigg`, `\Big`, etc. instead of `\left` and `\right`.

**Reply by HughGrovesArup** [reply]
Am I correct that with those commands the bracket would not scale to suit the contents though?

**Reply by edemaine** [reply]
Correct: these are manual sizing commands.

**Reply by HughGrovesArup** [reply]
OK, thanks, I will try and work something out within those constraints.

**Comment by edemaine** [comment]
Another possible workaround based on [this StackOverflow post](https://tex.stackexchange.com/a/21291/245104): you can manually split your equation at various points via `\right. \allowbreak \left.` and use `\vphantom` to get the automatic sizing to work.  More annoying, but more automatic sizing.
